File: 01.JAVA/pg-226-230/pg_228_24.java
import java.util.Scanner;

public class pg_228_24
{
    public static void main(String[] args)
        {
            Scanner enter = new Scanner(System.in);
            int num;

            System.out.println("");
            System.out.println("********************************************************");
            System.out.println("READ AN INTEGER AND DETERMINE THE SUM OF ITS EVEN DIGITS");
            System.out.println("********************************************************");
            System.out.println("");

            System.out.print("Write an integer: ");
            num = enter.nextInt();

            if(num < 0)
            {
                num = num * (-1);
            }

            int dig;
            int count = 0;

            while(num > 0)
            {
                dig = num % 10;
                num /= 10;

                if(dig % 2 == 0)
                {
                    count += dig;
                }
            }
            System.out.println("The sum of the even digits is " + count);
    }
}

File: 02.PYTHON/pg-226-230/pg_228_24.py
print("")
print("*********************************************************")
print("WRITE AN INTEGER AND DETERMINE THE SUM OF ITS EVEN DIGITS")
print("*********************************************************")
print("")

num = int(input("Write an integer: "))

if (num < 0):
    num = num * (-1)

dig = 0
count = 0

while(num > 0):
    dig = num % 10
    num //= 10

    if(dig % 2 == 0):
        count += dig
        
print("The sum of the even digits is " , count)